app/controllers/draws_controller.rb

```diff
@@ -37,6 +37,10 @@ def activate
     handle_action(action: 'show', **result)
   end
 
+  def intent_report
+    @students = @draw.students.order(:intent)
```

Comment:
Agreed, but I'm anticipating that we'll ultimately abstract out the whole reporting thing - this was just to get the routes in place and the initial template.